internal/viridian/cluster_log.go

```diff
@@ -16,12 +16,23 @@ func (a API) DownloadClusterLogs(ctx context.Context, destDir string, idOrName s
 	if err != nil {
 		return err
 	}
-	zipPath, stop, err := download(ctx, makeUrl(fmt.Sprintf("/cluster/%s/logs", c.ID)), a.token)
+	type resp struct {
+		path string
+		stop func()
+	}
```

Comment:
Would it make sense to introduce a generic Tuple type instead?
Something like the following  in the `internal/types` package?
```go
type Tuple2[A, B any] struct {
	First A
	Second B
}
```

Reply:
Fixed in: https://github.com/hazelcast/hazelcast-commandline-client/pull/300/commits/d3d3b8eb6da57e1de512a87e7ebd5597a1c31c6f